### DISCUSSION TITLE
Import generation data

### DISCUSSION THREAD

**Discussion by olivierlacan** [Stability-AI/StableSwarmUI#62] [Ideas]
As of now I'm not finding any way to import generation data as you might in Automatic1111. Is that something you're planning, or is it possible with an extension? 

Talking about allow folks with something like this in their pasteboard to paste it in the prompt UI somehow and have it recognized and automatically fill the relevant fields: 

```
<lora:Mycelium:1>cinsdia,building
Negative prompt: (worst quality, low quality:1.4), (bad_prompt_version2:0.8), EasyNegative, badhandv4, text, name, letters, watermark
Steps: 20, Size: 512x512, Seed: 3735051176, Model: realisticVisionV51_v51VAE, Sampler: DPM++ 2S a Karras, CFG scale: 7, Clip skip: 2, Model hash: 15012c538f, Hires steps: 20, Hires upscale: 2, Hires upscaler: R-ESRGAN 4x+ Anime6B, Denoising strength: 0.5
```

The kicker for me is Automatic1111 can't (AFAIK) auto-switch models based on this data, which seems more likely to be possible in Swarm because you don't instantly try to activate models when switching. Could be wrong but it seems like they're lazy loaded.

**Comment by mcmonkey4eva** [comment]
Oh, sorry for missing this for so long - please post an issue https://github.com/Stability-AI/StableSwarmUI/issues so I can make it as a feature request for this (though yes it is also in my personal todolist to happen eventually. An issue for it will bump the priority a bit at least)
